packages/@react-native-windows/cli/src/runWindows/runWindows.ts

```diff
@@ -142,7 +142,7 @@ async function runWindows(
       );
 
       newError(
-        `Please install the necessary dependencies by running ${rnwDependenciesPath} from an elevated PowerShell prompt.\nFor more information, go to http://aka.ms/rnw-deps`,
+        `It is possible your installation is missing depdencies. These can be automatically installed by running ${rnwDependenciesPath} from an elevated PowerShell prompt.\nFor more information, go to http://aka.ms/rnw-deps`,
```

Comment:
This is better, should we make it even clearer that these are installed dependencies to disambiguate between npm dependenices or nuget dependencies i.e.

`It is possible your computer does have all the necessary software installed.`

Reply:
Tweaked the wording a bit to try to make this clearer